easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb

```diff
@@ -11,7 +11,7 @@ toolchain = {'name': 'dummy', 'version': ''}
 
 source_urls = ['http://tandem.bu.edu/trf/downloads']
 sources = ['trf%(version_major)s%(version_minor)s.legacylinux64']
-checksums = [('md5', 'f97b6d1fc06adbd1484b384d5481916a')]
+checksums = ['e0221007bb58abb1c8609fe22adc75741c1b0de8d3943b4b40de407e1e2ddd57']
```

Comment:
@tutufan SHA256 checksum for my download (dating from May 30th 2016) is `6b7b1f23d89c91280ab31f972318a84a793b92e6559405a970c4459ce0e3ca3c`.

Did the binary change upstream, or did something go wrong with determining this checksum?

Reply:
I'm pretty sure the upstream just replace the tarball without bumping the version number.  Since Travis also built it, I'm assuming it's not an issue with EB checksumming.

Alternatively, I suppose it could be tainted by malware.  Probably just bad upstream engineering practices.

Reply:
I'm getting the same `SHA256` as you with a fresh download, and the timestamp for `trf409.legacylinux64` at http://tandem.bu.edu/trf/downloads/ suggests that the file was last updated Sept 20th 2016, which seems to explain this.

I'll archive my old download and replace it, thanks for clarifying!